ci/scripts/e2e-source-test.sh

```diff
@@ -159,6 +159,10 @@ risedev slt './e2e_test/source/basic/*.slt'
 risedev slt './e2e_test/source/basic/old_row_format_syntax/*.slt'
 risedev slt './e2e_test/source/basic/alter/kafka.slt'
 
+echo "--- e2e, kafka rate limit"
+risedev slt './e2e_test/source/basic/rate_limit_source_kafka.slt'
+risedev slt './e2e_test/source/basic/rate_limit_table_kafka.slt'
```

Comment:
Since they are new tests, and don't rely on any setup scripts, can you put the tests in `e2e_test/source_inline`? So that we don't need this new section in the sh, and easier to run them locally.

Reply:
You can use `${RISEDEV_KAFKA_WITH_OPTIONS_COMMON}` to replace `properties.bootstrap.server = 'message_queue:29092', connector = 'kafka'`

Reply:
Outdated these files are removed.